### DISCUSSION TITLE
create a html-button-response with clickable images

### DISCUSSION THREAD

**Discussion by valentinavellani** [jspsych/jsPsych#1118] [Q&A] [answered]
Hi All,
I was wondering if there is a way to create a  html-button-response in which the buttons are clickable images.

Thank you very much for your help!
Best
Valentina

**Comment by becky-gilbert** [comment, marked as answer]
Hi @valentinavellani, yes! The `choices` parameter is an array that contains the text for each button option, but you can put your image path(s) in that array instead. Then you'll want to use the `button_html` parameter to modify the default button HTML so that each "choice" is inserted as the "src" in an img tag. See the [html-button-response docs](https://www.jspsych.org/plugins/jspsych-html-button-response/#parameters) for more details about how the `button_html` parameter works. 

Here's an example:
```javascript
  var trial = {
    type: "html-button-response",
    stimulus: "",
    choices: ['jspsych-6.1.0/examples/img/blue.png','jspsych-6.1.0/examples/img/orange.png'],
    button_html: '<button class="jspsych-btn"><img src="%choice%"></button>',
    prompt: ""
  }
```

Also, because you're using image files that aren't being passed directly as trial parameters, you'll want to [manually preload the images](https://www.jspsych.org/overview/media-preloading/#manual-preloading).

```javascript
  jsPsych.init({
    timeline: [trial],
    preload_images: ['jspsych-6.1.0/examples/img/blue.png','jspsych-6.1.0/examples/img/orange.png']
  });
```

**Reply by valentinavellani** [reply]
Thank you very much Becky!
